From: Robin Snyders <robin@snyders.xyz>
Date: Wed, 12 Aug 2026 00:00:00 +0200
Subject: [PATCH] arm64: dts: qcom: sm8150: describe the sensor clock controller

Describe the block at 0x2b10000 so the sensor DSP can bring up the serial
engines it owns.

Signed-off-by: Robin Snyders <robin@snyders.xyz>
---
diff --git a/arch/arm64/boot/dts/qcom/sm8150.dtsi b/arch/arm64/boot/dts/qcom/sm8150.dtsi
index fbcc088..43bce60 100644
--- a/arch/arm64/boot/dts/qcom/sm8150.dtsi
+++ b/arch/arm64/boot/dts/qcom/sm8150.dtsi
@@ -2078,6 +2078,14 @@
 			status = "disabled";
 		};
 
+		scc: clock-controller@2b10000 {
+			compatible = "qcom,sm8150-scc";
+			reg = <0 0x02b10000 0 0x30000>;
+			clocks = <&rpmhcc RPMH_CXO_CLK>,
+				 <&sleep_clk>;
+			#clock-cells = <1>;
+		};
+
 		ufs_mem_hc: ufshc@1d84000 {
 			compatible = "qcom,sm8150-ufshc", "qcom,ufshc",
 				     "jedec,ufs-2.0";
